src/manual/testapi.adoc

```diff
@@ -0,0 +1,380 @@
+[[testapi]]
+= Test API Generation
+
+== OpenAPI: A Standard for API Description
+OpenAPI, formerly known as Swagger, is a widely adopted standard for describing RESTful APIs. It provides a
```

Comment:
I usually do one sentence per line in `*.adoc`. that makes it much more comparable in later pull-requests. would you mind giving it a refactoring? 👼🏼 sorry...

Reply:
It's currently not really done that way but i changed my code respectively.